app/views/messaging/partial_wrapper.scala.html

```diff
@@ -21,13 +21,31 @@
 
 @this(mainTemplate: gdsMainTemplate, govukAccordion : GovukAccordion, govukBackLink : GovukBackLink)
 
-@(partial: HtmlFormat.Appendable)(implicit request: Request[_], messages: Messages)
+@(partial: HtmlFormat.Appendable, titleKey: String)(implicit request: Request[_], messages: Messages)
 
 @mainTemplate(
- title = Title("conversation.heading"),
- withNavigationBanner = true
+  title = Title(titleKey),
+  useCustomContentWidth = true,
+  withNavigationBanner = true
 ) {
- <div class="govuk-grid-column-full">
-  @partial
- </div>
+  <div class="govuk-grid-row">
+
+    <div class="govuk-grid-column-two-thirds">
+      @partial
+    </div>
+
+    <div class="govuk-grid-column-one-third">
+      <div style="background-color:#eee; padding: 15px;">
+        <h4 class="govuk-heading-s">@messages("greyBox.heading")</h4>
+
+        <p class="govuk-body">@messages("greyBox.paragraph.1")</p>
+        <p class="govuk-body">@messages("greyBox.paragraph.2")</p>
+
+        <a class="govuk-link govuk-link--no-visited-state" href="@routes.MrnEntryController.onPageLoad">
+          @messages("greyBox.uploadFiles")
+        </a>
+      </div>
+    </div>
```

Comment:
instead of merging the 'conversation_wrapper' and the 'reply_result_wrapper' views into one common view perhaps it would be better to create a new component for this grey box that could be reused in both those views. If/when the conversation page and success page diverge from each other it will be easier to specialise these pages and imo would better aid discoverability.

Reply:
If you do not mind I'd leave it as it is. As you know I always try to implement according to the current requirements, and atm the 2 pages differ only for the partial. We can always do that **IF** they will actually diverge at some point. The partial is not so that complicated that even picking it up later you won't get whats for.